languages/shop-standards-de_DE.po

```diff
@@ -411,5 +411,9 @@ msgstr "Herr"
 msgid "Company"
 msgstr "Firma"
 
+#: src/WooCommerceSalutation.php:68
+msgid "Divers"
+msgstr "Diverse"
```

Comment:
This is reversed - "Diverse" is the English term, "Divers" is the German.  The code should contain the English term.

That said, in English, the term "Other" or "Intersex" may be used more frequently:
https://dict.leo.org/forum/viewUnsolvedquery.php?idThread=1365601&idForum=1&lp=ende&lang=de#followup=4

This is relevant, because the string might have to be translated into other languages in the future.

Reply:
Updated in the last commit.